dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -655,6 +659,25 @@ public List<Workflow> userAppTools(@ApiParam(hidden = true) @Parameter(hidden =
         return appTools;
     }
 
+    @GET
+    @Path("/{userId}/notebooks")
+    @Timed
+    @UnitOfWork
+    @Operation(operationId = "userNotebooks", description = "List all notebooks owned by the authenticated user.", security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME), method = "GET")
+    @ApiResponse(responseCode = HttpStatus.SC_OK
+        + "", description = "A list of notebooks owned by the user", content = @Content(array = @ArraySchema(schema = @Schema(implementation = Workflow.class))))
```

Comment:
Seems intentional to return Workflow, curious what was the reasoning to not return a Notebook from an explicitly Notebook-related endpoint?

Reply:
The analog `userServices` returns `List<Workflow>`, so I figured there was probably a reason and did the same thing here.  https://github.com/dockstore/dockstore/blob/b602fd29f6a9fb4bbcd61970dbc149017d4710e4/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java#L686